Automattic/wp-calypso
# Move generic stream component from reader/following-stream to reader/stream

To date, `reader/following-stream` has been the generic component that powered all Reader streams (tag stream, following, list stream) but also directly renders Following (/ or /following).

This PR splits them up, so the generic stream components are in reader/stream, and Following is rendered by reader/following/main.jsx.

### To test

Pull up all Reader streams (Following, Tag Stream, List Stream, Site Stream, Feed Stream) and ensure they all continue to function correctly.

Test live: https://calypso.live/?branch=fix/reader/move-following-stream
